### ISSUE TITLE
Project collection visibility can't be toggled with keyboard

### ISSUE THREAD

**Issue by SMaxOwok** [ManifoldScholar/manifold#2017]
>These buttons are keyboard inaccessible

These are just icons with click handlers.  Should be buttons
